config/application.rb

```diff
@@ -10,6 +10,7 @@
 
 module TPS
   class Application < Rails::Application
+    config.load_defaults 5.0
```

Comment:
J'ai l'impression que cette ligne peut nous embêter : la version 5.0 spécifie des valeurs qui ne sont pas celles qu'on a.

Si je comprends bien le plan de migration proposé par rails:
 - on spécifie les valeurs par défaut via `load_defaults` ([doc ici](https://guides.rubyonrails.org/configuring.html#results-of-config-load-defaults))
 - on monte jusqu'en 5.2, en surchargeant la config si besoin
 - quand c'est fait, on active un par un les paramètres de rails 6 en décommentant des lignes dans ` config/initializers/new_framework_defaults_6_0.rb `
 - quand tout est au vert, on supprime ` config/initializers/new_framework_defaults_6_0.rb ` et on met `    config.load_defaults 6.0`

Chez nous, aucun des paramètres décrit dans la doc concernant rails 5.0 n'est précisé. Il faudrait peut-êtr surcharger ces valeurs pour que ça corresponde à ce qu'on a actuellement pour être sur qu'il n'y ait pas de nouveauté, c'est-à-dire:

```ruby
config.action_controller.per_form_csrf_tokens: nil
config.action_controller.forgery_protection_origin_check: nil
ActiveSupport.to_time_preserves_timezone: false
config.active_record.belongs_to_required_by_default: false
config.ssl_options: {}
```

Reply:
(du coup ça peut valoir le coup d'avoir des fichiers de conf `new_framework_defaults_5_0.rb`, `new_framework_defaults_5_1.rb`etc pour rester dans la logique introduite mais je sens que je m'emporte un peu)

Reply:
On pourrait juste consolider tous les nouveaux défauts dans un fichier ?

Reply:
Sur le principe je suis pour l'explicite dans le fichier de config, oui 👍 

En revanche les réglages proposés ne correspondent pas à ceux qu'on a en ce moment chez nous :

```
$ bin/rails console
irb(main):002:0> config.action_controller.per_form_csrf_tokens
=> true
irb(main):003:0> config.action_controller.forgery_protection_origin_check
=> true
irb(main):004:0> ActiveSupport.to_time_preserves_timezone
=> true
irb(main):005:0> config.ssl_options
=> {:hsts=>{:subdomains=>true}}
```

Reply:
Ah mais n'importe quoi, j'ai regardé les défaults de Rails 6… 🤦

Reply:
@Keirua c'est bon, j'ai intégré ton commit.